space-cli/utils/file.go

```diff
@@ -34,10 +38,119 @@ func AppendConfigToDisk(specObj *model.SpecObject, filename string) error {
 	// Create a new file with out specs
 	return ioutil.WriteFile(filename, data, 0755)
 }
-func fileExists(filename string) bool {
+
+// ReadSpecObjectsFromFile returns the spec objects present in the file
+func ReadSpecObjectsFromFile(fileName string) ([]*model.SpecObject, error) {
+	var specs []*model.SpecObject
+
+	// Read the file first
+	data, err := ioutil.ReadFile(fileName)
+	if err != nil {
+		return nil, err
+	}
+
+	// Split the files into independent objects
+	dataStrings := strings.Split(string(data), "---")
```

Comment:
there was some array manipulation here, why did u remove it?

Reply:
It was incorrect. He was ignoring the last 3 characters assuming a file will always end with `---`. Thats not the case. I've handled the error this might cause below